Sources/PointFree/AtomFeed.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+import Html
+import HttpPipeline
+import MediaType
+import Prelude
+
+let atomFeedResponse =
+  writeStatus(.ok)
+    >-> respond(pointFreeFeed, contentType: .html)
```

Comment:
is the content type supposed to be xml?

Reply:
It is! Or `application/atom+xml`/`application(.atom(.xml))` Needs to be fixed.

Reply:
`MediaType` should prob be switched to a a struct huh? or at least have an `other` case?